deployer/scripts/upgrade.sh

```diff
@@ -46,6 +46,12 @@ function getDeploymentVersion() {
     return
   fi
 
+  # check for common data model
+  if [[ -n "$(oc get configmap/logging-elasticsearch -o yaml | grep 'use_common_data_model: true')" ]]; then
```

Comment:
@portante you are right - version 6 means "has es 2.x but does not have common data model"